pages/base/templatechecker.html

```diff
@@ -0,0 +1,29 @@
+/**
+* -----------------------------------------------------------------------------
+* @package     smartVISU
+* @author      Thomas Ernst
+* @copyright   2016
+* @license     GPL [http://www.gnu.de]
+* -----------------------------------------------------------------------------
+*/
+
+
+{% extends "system.html" %}
+
+{% block content %}
+<link rel="stylesheet" href="/lib/templatechecker/templatechecker.css">
+
+<h1>Template Checker: Page set "{{ config_pages }}"</h1>
+<div>
+	<div style="width:250px; float:left;"><input type="button" value="Run all tests" id="run"/></div>
+	<div style="width:250px; float:left;"><input type="button" value="Reset result" id="reset"/></div>
+	<div style="width:250px; float:left;"><input type="button" value="Show warnings/errors only" id="show_errors"/></div>
+	<div style="width:250px; float:left;"><input type="button" value="Show all messages" id="show_all"/></div>
+	<div style="clear:both"></div>
+</div>
+<h2>Result:</h2>
+<div id="filelist" style="display:none;"></div>
+
+<script src="/lib/templatechecker/templatechecker.js" type="text/javascript"></script>
```

Comment:
should be `<script src="lib/templatechecker/templatechecker.js" type="text/javascript">`